### ISSUE TITLE
Student Checkpoints

### ISSUE THREAD

**Issue by 2018cx** [brucewsy/AD-KD#1]
Hello, could you please advise me on how to obtain the reported scores for the qnli and qqp task? I utilized the pre-trained model available at https://huggingface.co/Brucewsy/AD-KD_bert_uncased_L-6_H-768_A-12_qnli and https://huggingface.co/Brucewsy/AD-KD_bert_uncased_L-6_H-768_A-12_qqp, but the resulting accuracy score on the development set was only 83.8 and 81.5. Could you kindly confirm if this Hugging Face model is the state-of-the-art model for this task? Additionally, are there any specific details that should be taken into consideration when testing these models?

**Comment by brucewsy** [comment]
I check the model checkpoints on HuggingFace and the acc of QNLI and QQP are consistent with the ones reported in the paper (91.163 on QQP, 91.232 on QNLI). The evaluation script is `python main_glue_distill.py --do_eval --do_lower_case --task_name qqp  --student_path [student_path] --teacher_path [teacher_path]`. Maybe you should check if there is a problem with your datasets, and set the option do_lower_case to True.

**Comment by 2018cx** [comment]
Thank you for your reply. I downloaded the datasets from https://dl.fbaipublicfiles.com/glue/data/QQP-clean.zip and https://dl.fbaipublicfiles.com/glue/data/QNLIv2.zip. Are these datasets the same as yours?

**Comment by brucewsy** [comment]
It seems same as the ones we use. Did you set max_seq_length=128 and use single GPU to evaluate?

**Comment by 2018cx** [comment]
I followed your evaluation script, set max_seq_length=128, and used a single GPU to evaluate. The QNLI model seems to be working well now, but the QQP model remains the same. I am wondering if the QQP dataset on the website has recently changed?

**Comment by brucewsy** [comment]
I evaluate on the QQP dataset from [https://dl.fbaipublicfiles.com/glue/data/QQP-clean.zip](url) and the acc is also 91.163.

**Comment by 2018cx** [comment]
Thank you for your help! I remove the previous preprocessed_dev_BertTokenizer_128 and now the model works well.
